android/sdk/src/main/java/app/organicmaps/sdk/location/MslAltitudeHelper.java

```diff
@@ -0,0 +1,60 @@
+package app.organicmaps.sdk.location;
+
+import static android.os.Build.VERSION.SDK_INT;
+
+import android.location.Location;
+import android.os.Build;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+public class MslAltitudeHelper
+{
+  public record AltitudeAndAccuracy(double altitude, float accuracy) {}
+  public record SpeedAndAccuracy(float speed, float accuracy) {}
+  public record BearingAndAccuracy(float bearing, float accuracy) {}
```

Comment:
So, this class provides not only altitude, right?

Reply:
Renamed to LocationCompatExtractor. Do you have a better name in mind?